frontend/packages/dev-console/src/components/add/AddPageLayout.tsx

```diff
@@ -25,58 +28,75 @@ const AddPageLayout: React.FC<AddPageLayoutProps> = ({ title, hintBlock: additio
-                    : t('devconsole~Hide add card details')
-                }
-                isChecked={showDetails}
-                onChange={(checked) => {
-                  setShowDetails(checked);
-                }}
-                data-test-id="odc-add-page-details-switch"
-              />
-            </Tooltip>
-            <span className="odc-add-page-layout__hint-block__details-switch__text">
-              {showDetails ? t('devconsole~Details on') : t('devconsole~Details off')}
-            </span>
+  const getHint = (): React.ReactNode => {
+    const hintText: string = t(
+      'devconsole~Select a way to create an Application, component or service from one of the options.',
+    );
+    const switchText: string = showDetails
+      ? t('devconsole~Details on')
+      : t('devconsole~Details off');
+    return (
+      <>
+        <div className="odc-add-page-layout__hint-block">
+          <div className="odc-add-page-layout__hint-block__text">{hintText}</div>
+          <div className="odc-add-page-layout__hint-block__actions">
+            <RestoreGettingStartedButton userSettingsKey={GETTING_STARTED_USER_SETTINGS_KEY} />
+            <div
+              className={cx('odc-add-page-layout__hint-block__details-switch', {
+                'odc-add-page-layout__hint-block__details-switch__loading-state': !isLoaded,
+              })}
+            >
+              {isLoaded ? (
+                <Tooltip
+                  content={t('devconsole~Show or hide details about each item')}
+                  position="top"
+                >
+                  <Switch
+                    aria-label={
+                      showDetails
+                        ? t('devconsole~Show add card details')
+                        : t('devconsole~Hide add card details')
+                    }
+                    isChecked={showDetails}
+                    onChange={(checked) => {
+                      setShowDetails(checked);
+                    }}
+                    data-test-id="odc-add-page-details-switch"
+                  />
+                </Tooltip>
+              ) : (
+                <Skeleton shape="circle" width="24px" />
+              )}
+              <span className="odc-add-page-layout__hint-block__details-switch__text">
+                {isLoaded ? switchText : <Skeleton height="100%" width="64px" />}
+              </span>
+            </div>
           </div>
         </div>
-      </div>
-      {additionalHint && (
-        <div className="odc-add-page-layout__additional-hint-block">{additionalHint}</div>
-      )}
-    </>
-  );
+        {additionalHint && (
+          <div className="odc-add-page-layout__additional-hint-block">{additionalHint}</div>
+        )}
+      </>
+    );
+  };
 
   return (
     <div className="odc-add-page-layout">
-      <PageLayout title={title} hint={hint} isDark>
+      <PageLayout title={title} hint={getHint()} isDark>
```

Comment:
```suggestion
      <PageLayout title={title} hint={getHint()}>
```

Please drop the `isDark` here. It only adds a light background to the page content which doesn't match the page itself. See:

![image](https://user-images.githubusercontent.com/139310/119469786-fa9f3e80-bd47-11eb-900e-226bc4535997.png)